app/src/main/java/org/fossasia/openevent/general/event/EventsListAdapter.kt

```diff
@@ -57,7 +59,7 @@ class EventsListAdapter(
      * The function to call when the adapter has to be cleared of items
      */
     fun clear() {
-        this.submitList(emptyList())
+//        this.submitList()
```

Comment:
how to pass empty paged list I couldn't get it

Reply:
What is this `emptyList()` method returning earlier?

Reply:
empty list returns a list whereas the adapter required a pagedlist

Reply:
thinking to use     
` this.currentList?.clear() `
would that work @liveHarshit